downloads/swan-lake-release-notes/swan-lake-beta4/RELEASE_NOTE.md

```diff
@@ -0,0 +1,831 @@
+---
+layout: ballerina-left-nav-release-notes
+title: Swan Lake Beta4
+permalink: /downloads/swan-lake-release-notes/swan-lake-beta4/
+active: swan-lake-beta4
+redirect_from: 
+    - /downloads/swan-lake-release-notes/swan-lake-beta4
+    - /downloads/swan-lake-release-notes/
+    - /downloads/swan-lake-release-notes
+---
+## Overview of Ballerina Swan Lake Beta4
+
+<em>This is the fourth Beta release in a series of planned Alpha and Beta releases leading up to the Ballerina Swan Lake GA release.</em> 
+
+It introduces the new language features planned for the Swan Lake GA release and includes improvements and bug fixes done to the compiler, runtime, standard library, and developer tooling after the Swan Lake Beta3 release.
+
+## Updating Ballerina
+
+If you are already using Ballerina, you can use the [Update Tool](/learn/tooling-guide/cli-tools/update-tool/) to directly update to Ballerina Swan Lake Beta4 as follows. 
```

Comment:
`Update Tool` link seems to be broken. Can you please check?

Reply:
Fixed.